python/idsse_common/idsse/common/validate_schema.py

```diff
@@ -13,8 +13,9 @@
 from typing import Optional, Union
 
 # pylint: disable=no-name-in-module
```

Comment:
Is this pylint disable still needed?

Reply:
I moved it to only suppress the warning for one line, but it seems yes it's needed. Without it, pylint warns that `RefResolver` doesn't exist (but the code runs fine?)

According to jsonschema docs, RefResolver isn't used anymore and we're supposed to use [a completely different approach](https://python-jsonschema.readthedocs.io/en/stable/referencing/#migrating-from-refresolver). I took a stab at migrating, but it seems to be a non-trivial change.